eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -177,8 +176,6 @@ function UpdateDocIndexFiles {
     # Update main.js var lang
     $mainJsContent = Get-Content -Path $MainJsPath -Raw
     $mainJsContent = $mainJsContent -replace "var SELECTED_LANGUAGE = ''", "var SELECTED_LANGUAGE = '$lang'"
-    # Update main.js var index html
-    $mainJsContent = $mainJsContent -replace "var INDEX_HTML = ''", "var INDEX_HTML = '$indexhtmlloc'"
```

Comment:
Don't you need to either set or remove https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/docgeneration/templates/matthews/styles/main.js#L8?

Reply:
Accidentally made the changes into .NET pr